README.md

```diff
@@ -180,103 +192,67 @@ module.exports = {
       }
     ]
   }
-  posthtml: function () {
+  posthtml: (ctx) => {
     return {
-      defaults: [ bem, each ]
+      plugins: [require('posthtml-include')({ root: ctx.resourcePath })]
+      parser: require('sugarml')
     }
   }
 }
 ```
-with custom package
 
-```js
-const bem = require('posthtml-bem')()
-const each = require('posthtml-each')()
-const include = require('posthtml-include')()
-
-module.exports = {
-  module: {
-    loaders: [
-      {
-        test: /\.html$/,
-        loader: 'html!posthtml?pack=html'
-      }
-    ]
-  }
-  posthtml: function () {
-    return {
-      defaults: [ bem, each ],
-      html: [ bem, each, include ]
-    }
-  }
-}
-```
 ## PostHTML with Jade engine in Express
 Also it's work with other view engine. Callback in `app.engine` is called by `res.render()` to render the template code.
 
 ```js
-app.engine('jade', function (path, options, callback) {
-    // PostHTML plugins
-    let plugins = [
-        require('posthtml-bem')(),
-        require('posthtml-textr')({ locale: 'ru'}, [
-            require('typographic-ellipses'),
-            require('typographic-single-spaces'),
-            require('typographic-quotes')
-        ])
-    ];
-
-    let html = require('jade').renderFile(path, options);
-
-    posthtml(plugins)
-        .process(html)
-        .then(function (result) {
-            if (typeof callback === 'function') {
-                var res;
-                try {
-                    res = result.html;
-                } catch (ex) {
-                    return callback(ex);
-                }
-                return callback(null, res);
-            }
-        });
+app.engine('jade', (path, options, cb) => {
+  const plugins = [
+    require('posthtml-bem')(),
+    require('posthtml-textr')({ locale: 'ru'}, [
+      require('typographic-ellipses'),
+      require('typographic-single-spaces'),
+      require('typographic-quotes')
+    ])
+  ]
+
+  let html = require('jade').renderFile(path, options)
+
+  posthtml(plugins)
+    .process(html)
+    .then((result) => {
+      if (typeof cb === 'function') {
+        let res
+
+        try {
+          res = result.html
+        } catch (ex) {
+          return cb(ex)
+        }
+          return cb(null, res)
+        }
+    })
 })
-app.set('view engine', 'jade');
+
+app.set('view engine', 'jade')
 ```
 
 ## Middleware
 
-<img align="right" height="56" title="KoaJS" src="http://t2.gstatic.com/images?q=tbn:ANd9GcRfnGHcTYGyMNcicU4N3nzV-5Rta9s_e5LzSI2HBjKMsLHundtmqAlQ" />
-
-### [Koa](http://koajs.com/)
+### [Koa](https://github.com/posthtml/koa-posthtml) <img align="right" height="56" title="Koa" src="http://t2.gstatic.com/images?q=tbn:ANd9GcRfnGHcTYGyMNcicU4N3nzV-5Rta9s_e5LzSI2HBjKMsLHundtmqAlQ" />
 
-##### [koa-posthtml](https://github.com/michael-ciniawsky/koa-posthtml)
+### [Hapi](https://github.com/posthtml/hapi-posthtml) <img align="right" height="56" title="Hapi" src="https://worldvectorlogo.com/logos/hapi.svg" />
```

Comment:
![text](https://cloud.githubusercontent.com/assets/5419992/17646634/d19f6c06-61d2-11e6-92c9-4849e338d43f.png)

```hspace="50%"``` should move to the left like in the screenshot above, for github preview only 

![text_hspace](https://cloud.githubusercontent.com/assets/5419992/17646637/e8168d98-61d2-11e6-8a0a-243958cb252c.png)

![logo](https://cloud.githubusercontent.com/assets/5419992/17646638/eef3e304-61d2-11e6-9d55-ae2725cf3dd2.png)

```hspace="50%"``` should move to the left like in the screenshot above, for github preview only 

![logo_hspace](https://cloud.githubusercontent.com/assets/5419992/17646639/f77308b6-61d2-11e6-8f58-861918caa3f7.png)

Choose what you like, or if disliked in general we use just the headings :)